packages/odie-client/src/constants.ts

```diff
@@ -7,6 +7,11 @@ export const ODIE_ERROR_MESSAGE = __(
 	__i18n_text_domain__
 );
 
+export const ODIE_ERROR_MESSAGE_NON_ELIGIBLE = __(
+	'Sorry, I am offline right now. Come back later or ask for help in our forums.',
```

Comment:
A small suggestion to be consistent with a similar message, and to visually connect the message to the button:

```suggestion
	'Sorry, I am offline right now. Come back later or ask for help in our forums:',
```